docs/usage/getting-started/running.md

```diff
@@ -81,9 +83,10 @@ The Renovate team provide a ["Renovate Runner"](https://gitlab.com/renovate-bot/
 This supports both `gitlab.com` and self-hosted GitLab.
 Details for how it works can be found in the project.
 
-#### Mend Renovate On-Premises
+#### Mend Renovate Community Edition / Enterprise Edition
 
-Mend Renovate On-Premises (WSOP) started out as a commercial product "Renovate Pro", but was renamed and made free to use when Renovate became a part of Mend (formerly WhiteSource) in 2019.
+Mend Renovate Community Edition (Renovate CE) and Enterprise Edition (Renovate EE) are non-Open Source offerings of Renovate for self-hosted users.
+Renovate CE and Renovate EE have support for GitHub (both `github.com` and GitHub Enterprise Server) as well as GitLab self-hosted.
```

Comment:
```suggestion
You can run Renovate CE and Renovate EE on GitHub (both `github.com` and GitHub Enterprise Server) or on a self-hosted GitLab instance.
```

How about something like this?

Reply:
Thanks for the suggestion. I worded it this way because Renovate doesn't technically "run on" GitHub or GitLab. It just "accesses" repos on GitHub or GitLab. So it "has support for" GH and GL. I might keep this as it is for now.